resources/assets/components/PagingButtons/index.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+
+class PagingButtons extends React.Component {
+
+  render() {
+    const prev = this.props.prev;
+    const next = this.props.next;
+    const prevCopy = this.props.prevCopy;
+    const nextCopy = this.props.nextCopy;
+
+    return (
+        <div className="container__block">
+			<a href={prev}>{prevCopy}</a>
+			<div className="next-page">
```

Comment:
@DFurnes I think this is cleaner now too, it's easier to see that the text is `null` and nothing is displayed if there isn't a page to go to